cudax/include/cuda/experimental/__async/completion_signatures.cuh

```diff
@@ -27,28 +27,37 @@
 
 #include <cuda/experimental/__async/cpos.cuh>
 #include <cuda/experimental/__async/exception.cuh>
+#include <cuda/experimental/__async/type_traits.cuh>
 
 #include <cuda/experimental/__async/prologue.cuh>
 
 namespace cuda::experimental::__async
 {
 // A typelist for completion signatures
-template <class... _Ts>
+template <class... _Sigs>
 struct completion_signatures
-{};
+{
+  struct __partitioned;
+};
+
+template <class _CompletionSignatures>
+using __partitioned_completions_of = typename _CompletionSignatures::__partitioned;
+
+constexpr int __invalid_disposition = -1;
```

Comment:
Should that be 
```suggestion
inline constexpr int __invalid_disposition = -1;
```

Reply:
unless you care about the address of a thing being the same in all translation units, it doesn't really matter, i don't think.